configuration/pih/pih-config-liberia-harper-dev.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
This is a new development configuration I set up that we can use on our development laptops.  It has the exact same components as the "pih-config-liberia-harper", but also has MCH enabled so we can work on developing and testing that.

To use this, in your runtime properties you'd basically just set:
pih.config=liberia,liberia-harper,liberia-harper-dev

The components defined in "liberia-harper" would be overridden with the ones defined here.

Reply:
OK, that's good! That way we get the flexibility of not manually changing configs between environments. Is that correct @mogoodrich ?

Reply:
@BranfordTGbieor  it allows you to have a configuration setting you can use for development (of the MCH forms right now) which is different than what is on Kouka, so we can have Kouka show exactly what will go out to JJ Dossen.